xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
+    },
+    "bt_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Beamtime object"
+    },
+    "sample_name":{
+      "type": "string",
+      "description": "name of sample being measured"
+    },
+    "sample_composition":{
+      "type": "object",
+      "description": "dictionary of sample stoichiometry"
+    },
+    "sample_phase":{
+      "type": "object",
+      "description": "dictionary of sample phase information"
+    },
+    "sample_maker":{
+      "type": "array",
+      "description": "list of sample maker names "
+    },
+    "sa_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Sample object"
+    },
+    "cif_name":{
+      "type": "array",
+      "description": "list of possible structure filenames"
```

Comment:
Because a spread sheet is a great data-entry tool, but not a great data-preservation tool.  You want to capture the value of the meta-data at the time of data-collection, not what they changed it to later.